templates/auth.yaml

```diff
@@ -93,6 +91,16 @@ Resources:
                   - !Sub "arn:${AWS::Partition}:glue:*:${AWS::AccountId}:database*"
                   - !Sub "arn:${AWS::Partition}:glue:*:${AWS::AccountId}:table*"
                   - !Sub "arn:${AWS::Partition}:glue:*:${AWS::AccountId}:partition*"
+        - PolicyName: APIGateway
+          PolicyDocument: 
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "execute-api:Invoke"
+                  - "execute-api:ManageConnections"
+                Resource: !Sub arn:${AWS::Partition}:execute-api:${AWS::Region}:${AWS::AccountId}:*
```

Comment:
API Gatway ARNs do no have an account ID part. You can see the ARN format here: https://docs.aws.amazon.com/apigateway/latest/developerguide/arn-format-reference.html.